Vagrantfile

```diff
@@ -122,7 +117,7 @@ Vagrant.configure("2") do |config|
       ansible.verbose = 'v'
       # the production playbook verifies that staging default values are not
       # used will need to skip the this role to run in Vagrant
-      ansible.skip_tags = [ "validate" ]
+      ansible.skip_tags = ENV['PROD_SKIP_TAGS']
```

Comment:
Should this have `|| "validate"` appended to it, or are we no longer skipping the `validate` role for `*-prod` hosts?